Tests/TestProjectGenerator/Reqnroll.TestProjectGenerator/TargetFrameworkMonikerStringBuilder.cs

```diff
@@ -27,6 +27,7 @@ public class TargetFrameworkMonikerStringBuilder
             [TargetFramework.Net60] = "net6.0",
             [TargetFramework.Net70] = "net7.0",
             [TargetFramework.Net80] = "net8.0",
+            [TargetFramework.Net90] = "net9.0"
```

Comment:
```suggestion
            [TargetFramework.Net90] = "net9.0",
```

This will make adding a new framework a one-line diff in the future.